dockerfiles/init/modules/che-kubernetes-helm/templates/configmap.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: che
+  name: che
+data:
+  CHE_HOST: {{ template "cheHost" . }}
+  CHE_PORT: "8080"
+{{- if .Values.tlsEnabled }}
+  CHE_API: https://{{ template "cheHost" . }}/api
+  CHE_WEBSOCKET_ENDPOINT:  wss://{{ template "cheHost" . }}/api/websocket
+  CHE_INFRA_KUBERNETES_BOOTSTRAPPER_BINARY__URL: https://{{ template "cheHost" . }}/agent-binaries/linux_amd64/bootstrapper/bootstrapper
+{{ else }}
+  CHE_API: http://{{ template "cheHost" . }}/api
+  CHE_WEBSOCKET_ENDPOINT:  ws://{{ template "cheHost" . }}/api/websocket
+  CHE_INFRA_KUBERNETES_BOOTSTRAPPER_BINARY__URL: http://{{ template "cheHost" . }}/agent-binaries/linux_amd64/bootstrapper/bootstrapper
+{{- end }}
+  CHE_DEBUG_SERVER: "true"
+  CHE_INFRASTRUCTURE_ACTIVE: kubernetes
+  CHE_INFRA_KUBERNETES_MACHINE__START__TIMEOUT__MIN: "5"
+  CHE_INFRA_KUBERNETES_MASTER__URL: ""
+  CHE_INFRA_KUBERNETES_OAUTH__TOKEN: ""
+  CHE_INFRA_KUBERNETES_PASSWORD: ""
+  CHE_INFRA_KUBERNETES_USERNAME: ""
+  CHE_INFRA_KUBERNETES_NAMESPACE: ""
+  CHE_INFRA_KUBERNETES_TRUST__CERTS: "false"
+  CHE_INFRA_KUBERNETES_PVC_STRATEGY: "common"
+  CHE_INFRA_KUBERNETES_PVC_PRECREATE__SUBPATHS: "true"
+  CHE_INFRA_KUBERNETES_POD_SECURITY__CONTEXT_RUN__AS__USER: "0"
+  CHE_INFRA_KUBERNETES_POD_SECURITY__CONTEXT_FS__GROUP: "0"
+  CHE_LOCAL_CONF_DIR: /etc/conf
+  CHE_LOGS_DIR: /data/logs
+  CHE_LOG_LEVEL: "INFO"
+  CHE_MULTIUSER: "false"
+  CHE_OAUTH_GITHUB_CLIENTID: ""
+  CHE_OAUTH_GITHUB_CLIENTSECRET: ""
+  CHE_PREDEFINED_STACKS_RELOAD__ON__START: "false"
+  JAVA_OPTS: "-XX:MaxRAMFraction=2 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true -Xms20m "
+  CHE_WORKSPACE_AUTO_START: "false"
+  CHE_INFRA_KUBERNETES_INGRESS_ANNOTATIONS__JSON: '{"nginx.ingress.kubernetes.io/rewrite-target": "/","nginx.ingress.kubernetes.io/ssl-redirect": "false","nginx.ingress.kubernetes.io/proxy-connect-timeout": "3600","nginx.ingress.kubernetes.io/proxy-read-timeout": "3600"}'
```

Comment:
Should not this value be updated according to the value of  `.Values.tlsEnabled` configuration property?

Reply:
It should, but we will tackle TLS once we fully support host-based routing for workspace ingresses.

Reply:
OK then